### ISSUE TITLE
Not launching on Manjaro-ARM XFCE

### ISSUE THREAD

**Issue by LizzieA69** [buzz/twitch-indicator#15]
I use Manjaro-ARM XFCE on a Raspberry Pi 400. I just received the indicator that you have an update of your app ready. I quit out of the app and ran the updated. When I tried to launch it again, nothing happened. I then added it to the desktop and tried to launch it with the same results. I then tried rebuilding it and launching it and nothing. I did a restart of my Pi and tried to launch it and nothing happened. Previously, before the update, I had added it to my auto start so it would launch at startup of my Pi. I removed it from auto start then deleted it and restarted my Pi then reinstalled it. I went to launch it and nothing happened. What is going on with it? Please fix it so I can use it again. It is a very valuable tool for me.

**Comment by buzz** [comment]
That's probably due to #14. Make sure to have libnotify v0.8 installed. Did you do a full system upgrade, too? The new libnotify landed on Arch not long ago. Don't know about Manjaro though.

Also, please append a console log output, otherwise it's difficult to know what's happening.

**Comment by Keelhauled** [comment]
I'm getting this with libnotify 0.8.0-2 installed on arch
```
% twitch-indicator 
Traceback (most recent call last):
  File "/usr/bin/twitch-indicator", line 33, in <module>
    sys.exit(load_entry_point('twitch-indicator==1.4', 'gui_scripts', 'twitch-indicator')())
  File "/usr/bin/twitch-indicator", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 171, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/lib/python3.10/site-packages/twitch_indicator/__main__.py", line 6, in <module>
    gi.require_version("Notify", "0.8")
  File "/usr/lib/python3.10/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available for version %s' %
ValueError: Namespace Notify not available for version 0.8
```

**Comment by Keelhauled** [comment]
Okay I downgraded to 0.8.0-1 and it works

**Comment by flroux** [comment]
This fix https://gitlab.gnome.org/GNOME/libnotify/-/issues/27 seem to broke twitch-notifier 1.4 with libnotify 0.8.1

On Arch 0.8.0-2 include a patch with this fix

**Comment by buzz** [comment]
libnotify released v0.8.1 which reverts their mistake.

Upgrade libnotify + twitch-indicator to the newest versions and all should be back to normal now.
